tests/foreman/ui/test_host.py

```diff
@@ -367,6 +367,154 @@ def test_positive_create_libvirt(self):
             self.assertIsNotNone(search)
 
     @run_only_on('sat')
+    @stubbed
+    @tier3
+    def test_positive_create_baremetal_bios(self):
```

Comment:
Don't think that test cases like this are applicable for UI level. Could you please confirm that we will not do 99% of actions still through API or CLI?